lib/teracy-dev/util.rb

```diff
@@ -24,23 +22,26 @@ def self.exist?(value)
     # false, "false", "f", "no", "n", "0" => false
     # otherwise, error raised
     def self.boolean(value)
-      return true if value == true || value.to_s.downcase =~ (/(true|t|yes|y|1)$/i)
-      return false if value == false || value.to_s.empty? || value.to_s.downcase =~ (/(false|f|no|n|0)$/i)
+      true_regexp = /(true|t|yes|y|1)$/i
+
+      return true if value == true || value.to_s.downcase =~ true_regexp
+
+      false_regexp = /(false|f|no|n|0)$/i
+
+      return false if value == false || value.to_s.empty? || value.to_s.downcase =~ false_regexp
+
       raise ArgumentError.new("invalid value for boolean: #{value}")
     end
 
-
     # convert a value (case insentitive) to a boolen value
     # true, "true", "t", "yes", "y", "1" => true
     # othwerwise, false
     #
     # @since v0.6.0-a5
     def self.true?(value)
-      begin
```

Comment:
should have `begin`

Reply:
> should have `begin`

In Ruby 2.5 or later, you can omit `begin` in `do-end` block.

Reply:
we will change this in the future!

Reply:
So we need to force user to use ruby >=2.5 which is not necessary, my ruby version:
```
$ ruby -v
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin16]
```

Reply:
@datphan make sure to review the latest changes from the PR instead.